src/app/Main/Body/QueryPage/Results/Charts/QueryPageContent-BarChart.component.tsx

```diff
@@ -93,7 +94,7 @@ export function QueryPageContentBarChartResultsComponent(props: {
             props.columnAttributes
         );
 
-        if (_.isEmpty(processedChartData.series)) {
+        if (_.isEmpty(processedChartData.series) || props.chartIsLoading) {

```

Comment:
So data could be loaded and loadingProgress is still shown due to empty chart data? That might be confusing for the user.

Reply:
no it's not empty chart data. It is only there to bridge the seconds until changes are actually visible. The table has the same feature.

Reply:
So the use case is chartIsLoading is ready but processedChartData.series is still empty? Then it would make sense.

Reply:
The data is only empty if no diagram has been displayed yet. If you change the data, there is no time when they are empty. First the old data is stored and then the new data. I bridge the change between old and new data with the loading circle.

Reply:
So, if the page is loaded with no data the chart is empty and a spinner is shown? That would be the situation i targeted with my first comment at.